era_5g_interface/task_handler_internal_q.py

```diff
@@ -45,13 +67,40 @@ def store_image(self, metadata: dict, image: np.ndarray) -> None:
                 The format is NetApp-specific.
             image (_type_): The image to be processed.
         """
+
```

Comment:
please rename the method to "store_data" and the field image to data and put there Any as a anotation

Reply:
done